lib/gooder_data/project/report.rb

```diff
@@ -93,6 +94,14 @@ def get_url_report_export(fmt)
         end
       end
 
+      def download(url)
+        response = get(url)
```

Comment:
Se o `response` não for `202` no primeiro caso o que deveria acontecer?
Porque eu pedi pra fazer um download de um `.pdf`, quando eu abri deu a mensagem "_Failed to load PDF document_"